.travis.yml

```diff
@@ -40,10 +40,10 @@ jobs:
 #      script: mvn test
 #      after_success: mvn prepare-package -DskipTests=true
   allow_failures:
+    - os: osx
```

Comment:
This was our Java 14 test, so we should probably add a replacement to cover that.